platform/x11/context_gl_x11.cpp

```diff
@@ -83,6 +83,19 @@ static int ctxErrorHandler(Display *dpy, XErrorEvent *ev) {
 	return 0;
 }
 
+static void setClassHint(Display *display, Window window) {
```

Comment:
Please use `snake_case` for Godot methods, and prefix parameters with `p_` (so `p_display`, `p_window`).